test/functional/wallet_basic.py

```diff
@@ -273,6 +273,9 @@ def run_test(self):
         msg = "Invalid amount"
         for invalid_value in [0.00000001, 0.00099999, "0.00000001", "0.00099999"]:
             assert_raises_rpc_error(-3, msg, self.nodes[2].sendmany, amounts={address: 10}, fee_rate=invalid_value)
+        # Test fee_rate values rejected by ParseFixedPoint
```

Comment:
My cited comment was in reply to a question about documentation. Which coverage should be dropped? It would have certainly been easier to not add coverage, but where fee rate test coverage has been partial or absent over the series of fee rate changes, I've ended up bitten by it and prefer to have the coverage in place for these changes and for future ones. The documentation is helpful to me when coming back to this months later (like now) to regain context. When I'm done with the fee rate work, I'm happy to remove the documentation.

Reply:
Changed the comments in this commit to not name the specific function: "Test fee_rate values that don't pass fixed-point parsing checks".